test/specs/views/Card/CardGroup-test.js

```diff
@@ -0,0 +1,43 @@
+import faker from 'faker'
+import React from 'react'
+import * as common from 'test/specs/commonTests'
+import Card from 'src/views/Card/Card'
+import CardGroup from 'src/views/Card/CardGroup'
+
+describe('CardGroup', () => {
+  common.isConformant(CardGroup)
+  common.hasUIClassName(CardGroup)
+  common.propKeyOnlyToClassName(CardGroup, 'doubling')
+  common.propKeyOnlyToClassName(CardGroup, 'stackable')
+  common.implementsNumberToWordProp(CardGroup, 'itemsPerRow')
+
+  describe('renders children', () => {
+    const firstText = faker.hacker.phrase()
+    const secondText = faker.hacker.phrase()
+
+    it('with `children` prop', () => {
+      const wrapper = mount(
+        <CardGroup>
+          <Card>{firstText}</Card>
+          <Card>{secondText}</Card>
+        </CardGroup>
+      )
+        .find('Card')
+
+      wrapper.first().should.contain.text(firstText)
+      wrapper.last().should.contain.text(secondText)
+    })
+
+    it('with `items` prop', () => {
```

Comment:
This would then be a describe.  Good rule of thumb is one describe() per prop.

If there is no prop to describe the behavior, it is probably a top level `it()` assertion.  In some rare cases, there are special scenarios that need a `describe()` outside of this.